doc/source/whatsnew/v1.0.0.rst

```diff
@@ -166,6 +166,7 @@ Groupby/resample/rolling
 
 -
 -
+- Bug in :meth:`groupby` does not offer implicit selection by column names when ``axis`` set to 1 (:issue:`27614)
```

Comment:
Should be DataFrame.groupby`

"does not offer" to "not offering" 

"axis set to 1" to "`axis=1`"

Reply:
I'm also not sure what "implicit selection by column name" means here.

Reply:
> I'm also not sure what "implicit selection by column name" means here.

ahh, my bad, i basically just copied and pasted the issue title, will change! @TomAugspurger thanks for your quick reply!